src/App.js

```diff
@@ -1,12 +1,90 @@
 import React from 'react';
+import {useState, useEffect} from "react";
+import axios from 'axios';
+import Card from "./components/card/Card";
 import './App.css';
 
+
+
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [pokemonList, setPokemonList] = useState({});
+    const [pokemonUrl, setPokemonUrl] = useState('https://pokeapi.co/api/v2/pokemon/');
+
+    useEffect(
+        () => {
+            async function fetchData() {
+                let result = {};
+                //console.log("fetch data ");
+                try {
+                    result = await axios.get(pokemonUrl);
+                    //console.log(result.data);
+
+                    let list = {};
+                    list.count = result.data.count;
+                    list.next = result.data.next;
+                    list.previous = result.data.previous;
```

Comment:
Het initiëren en vullen van het object kun je in één keer doen. Verder overschrijf je het object niet, dus zou je een `const` kunnen gebruiken:
```
const list = {
    count: result.data.count,
    next: result.data.next,
    previous: result.data.previous
};
```